docs/accelerators.md

```diff
@@ -91,6 +101,8 @@ docker run --rm -it  --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/rende
 
 Support for [Intel Arc](https://www.intel.com/content/www/us/en/architecture-and-technology/visual-technology/arc-discrete-graphics.html), which is in preview now, requires newer driver version `22.10.22597`. You can build OpenVINO Model server with ubuntu base image and that driver using the command below:
 ```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
 make docker_build INSTALL_DRIVER_VERSION=22.10.22597
```

Comment:
@mzegla is it still relevant? We have `INSTALL_DRIVER_VERSION=dg2` now if I'm not mistaken?

Reply:
Not yet. The PR is not yet merged and the drivers for dGPU are not public. 
For integrated cards it's still true, but soon 22.10 should be default version and this parameter will not be necessary. 
But it's something we can change in another PR once we're sure.